src/openrct2/localisation/StringIds.h

```diff
@@ -3343,7 +3343,7 @@ enum : uint16_t
     STR_HEIGHT_MAP_ERROR = 6048,
     STR_ERROR_READING_PNG = 6049,
     STR_ERROR_READING_BITMAP = 6050,
-    STR_ERROR_WIDTH_AND_HEIGHT_DO_NOT_MATCH = 6051,
+    // STR_ERROR_WIDTH_AND_HEIGHT_DO_NOT_MATCH = 6051, // Unused
```

Comment:
Just delete the string and this entry.

Reply:
I went with commenting it out and annotating it because unexplained gaps can be difficult later on. This way, it at least explains why it's there.

https://github.com/OpenRCT2/OpenRCT2/blob/5f2bde9375a014d33c32e2f77e82efbfc488d1ff/src/openrct2/localisation/StringIds.h#L1754

https://github.com/OpenRCT2/OpenRCT2/blob/5f2bde9375a014d33c32e2f77e82efbfc488d1ff/src/openrct2/localisation/StringIds.h#L1863-L1864

https://github.com/OpenRCT2/OpenRCT2/blob/5f2bde9375a014d33c32e2f77e82efbfc488d1ff/src/openrct2/localisation/StringIds.h#L1893-L1894

On other places, it looks like strings are replaced with `<removed string - do not use>`, thought from what I can find these were strings from the original game:

https://github.com/OpenRCT2/OpenRCT2/blob/5f2bde9375a014d33c32e2f77e82efbfc488d1ff/src/openrct2/localisation/StringIds.h#L2006-L2008

https://github.com/OpenRCT2/OpenRCT2/blob/5f2bde9375a014d33c32e2f77e82efbfc488d1ff/src/openrct2/localisation/StringIds.h#L1845-L1850

With deleting the string, you mean from the `en-GB.txt` file I presume?

Reply:
@Gymnasiast Reminder

Reply:
@Gymnasiast second reminder. Can you please comment on whether you still want these removed or not? I'd like to get this PR merged.

Reply:
Yes, I still think they should be removed from both this file and en-GB. I don’t think gaps are that confusing, and commented-out code is smelly - and we have Git if we ever want to get these back.